stream-chat-swift-docs/docs/guides/getting-started.md

```diff
@@ -1,3 +1,149 @@
+
+#### Static
+
+```swift
+/// A static token provider. Use it for setting the token synchronously or for testing purposes.
+let token = Token("USER_TOKEN")
+let tokenProvider = TokenProvider.static(token)
+```
+
+#### Closure
+
+```swift
+/// A token provider that fetches the token from your backend service. Use it in production.
+let tokenProvider = TokenProvider.closure { chatClient, completion in
+    let token: Token?
+    let error: Error?
+
+    /// TODO: Fetch a token locally or use URLSession/Alamofire/etc to fetch
+    /// a token from your backend service and pass it into completion
+
+    if let token = token {
+        completion(.success(token))
+    } else if let error = error {
+        completion(.failure(error))
+    }
+}
+```
+
+#### Development
+
+```swift
+/// A development token provider. Use it for testing purposes.
+let tokenProvider = TokenProvider.development(userId: "USER_ID")
+```
+
+### Guest
+
+Guest users need to be identified, but they don't require server-side authentication. They're ideal for support and livestream use cases. You'll likely need to [configure permissions](https://getstream.io/chat/docs/node/chat_permission_policies/?language=js) as most interactions are disabled by default for guests.
+
+```swift
+/// A guest token provider. Use it to let the user interact with your chat before having a real account.
+let tokenProvider = TokenProvider.guest(userId: "USER_ID")
+```
+
+### Anonymous
+
+Anonymous users don't need a special token or any identification, but they can't do much except reading livestream chats. They're ideal for the livestream use case before the user wants to identify themselves and interact.
+
+```swift
+/// An anonymous token provider. Use it to let the user see livestream chats without identifying themselves or creating an account.
+let tokenProvider = TokenProvider.anonymous
+```
+
+## Configuration
+
+The next step is to configure a `ChatClient` instance with your API Key and the `tokenProvider` from the previous step. The most simple way to do this is by extending the `ChatClient` class with a shared instance that will be used throughout your app (Singleton). It's also possible to create a chat instance in a convenient entry point and pass it down to your classes / view controllers. (Dependency Injection).
+
+### ChatClientConfig
+
+The `ChatClientConfig` object holds properties that the chat client will use to determine certain behaviors. For example, the `apiKey`, which tells the chat client which chat server to communicate with. It also has the `baseURL` property which tells the chat client which region of the world your server is at, which can be useful to reduce overall latency.
```

Comment:
Needs a code snippet to show how `baseURL` is changed, preferably also showing the dashboard so users can configure the correct baseURL